module-1/lab-api-scavenger-game/your-code/challenge-1.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
Gooooood job Maris!! Just on thing, you should normalize the JSON object to avoid unexpected results!!